tests/cmake/CMakeLintBearTest.py

```diff
@@ -27,3 +27,11 @@
     invalid_files=(),
     tempfile_kwargs={'suffix': '.cmake'},
     settings={'cmakelint_config': conf_file})
+
+CMakeLintBearConfigTest = verify_local_bear(
```

Comment:
Better to choose different name as `CMakeLintBearConfigTest` is already taken. 😄